include/dxc/Support/dxcapi.extval.h

```diff
@@ -0,0 +1,20 @@
+#include "dxc/Support/dxcapi.use.h"
+#include "dxc/dxcapi.h"
+
+class DxcDllExtValidationSupport : public dxc::DxcDllSupport {
+  // DxcDllExtValidationSupport manages the
+  // lifetime of dxcompiler.dll, while the member, m_DxilSupport,
+  // manages the lifetime of dxil.dll
+  dxc::DxcDllSupport DxilSupport;
+
+  std::string DxilDllPath;
```

Comment:
Come to think of it, maybe we don't need to store the string.  We could use the string returned from getenv in InitializeInternal, and just keep a flag indicating whether we tried to initialize the external dll but failed.

Do we have an actual use for the `GetDxilDllPath()` accessor?

Reply:
Oh right, perhaps for the scenario were we want to print that path as part of diagnostics or the version info in DXC, or some other thing.

Reply:
Yes, so I'll keep the string stored for now.